data/categories.txt

```diff
@@ -8922,6 +8922,46 @@ vi:Quần vợt
 zh-Hans:网球
 zh-Hant:網球
 
+sport-padel
+en:Padel
```

Comment:
Should tennis be added as a synonym?

Reply:
somehow its possible to make "padel" match tennis places, but not possible to make "tennis" match padel places 🤪